src/asynk/async_worker.rs

```diff
@@ -0,0 +1,144 @@
+use crate::asynk::async_queue::AsyncQueue;
+use crate::asynk::async_queue::FangTaskState;
+use crate::asynk::async_queue::Task;
+use crate::asynk::async_runnable::AsyncRunnable;
+use crate::asynk::Error;
+use bb8_postgres::tokio_postgres::tls::MakeTlsConnect;
+use bb8_postgres::tokio_postgres::tls::TlsConnect;
+use bb8_postgres::tokio_postgres::Socket;
+use log::error;
+use std::thread;
+use std::time::Duration;
+use typed_builder::TypedBuilder;
+#[derive(Clone, Debug)]
+pub enum RetentionMode {
```

Comment:
since these configuration parameters are common with the blocking feature. Let's share them

Reply:
yes, or we can put it some module that won't be under any feature flags (common to every feature)